### DISCUSSION TITLE
How to config automatically reconnect

### DISCUSSION THREAD

**Discussion by webclient163** [SuRGeoNix/Flyleaf#476] [Q&A] [answered]
please tell me:What settings do I need to modify in order for the player to automatically reconnect after a 1-second when playing RTSP video streams but network disconnected?
Or how can we detect that the player has stopped playing?

**Comment by SuRGeoNix** [comment, marked as answer]
@webclient163 did you check this https://github.com/SuRGeoNix/Flyleaf/issues/240?

**Reply by webclient163** [reply]
Thank you!
